bluesky_config/ipython/profile_default/startup/00-base.py

```diff
@@ -47,18 +45,19 @@
     serializer=partial(msgpack.dumps, default=mpn.encode),
 )
 
-bec = BestEffortCallback()
-
 logger = logging.getLogger("databroker")
 logger.setLevel("DEBUG")
 handler = logging.StreamHandler()
 handler.setLevel("DEBUG")
 logger.addHandler(handler)
 
-RE.subscribe(db.v1.insert)
 RE.subscribe(zmq_publisher)
 RE.subscribe(kafka_publisher)
-RE.subscribe(bec)
+try:
+    bec = BestEffortCallback()
+    RE.subscribe(bec)
+except Exception:
```

Comment:
Is this moved inside try/except to support the headless use case? I would prefer a more explicit mechanism like:

```
podman run ...
       ...
       -e BLUESKY_HEADLESS=1 \
       ...
```

and, here,
```py
if os.getenv("BLUESKY_HEADLESS") != "1":
    bec = ...
```
because silently skipping if any exception is raised  could create hard-to-debug failure modes when the user _wants_ `bec`.

Reply:
Actually I think I can just remove this try and it will work for headless.

Reply:
Updated